data/tuyoData.json

```diff
@@ -4,6 +4,36 @@
             "quote": "Beside the coals of the dying fire, within the trampled borders of our abandoned camp, surrounded by the great forest of the winter country, I waited for a terrible death.",
             "chapter": "Chapter 1",
             "percentage": 1
+        },
+        {
+            "quote": "At last the warleader dismounted. He was far taller than I; even taller than most of his own people. He looked cruel to me, with a hard set to his mouth.",
+            "chapter": "Chapter 1",
+            "percentage": 1
+        },
+        {
+            "quote": "He said to that man, \"We must have pressed them even harder than we knew, if they've left a tuyo for us.\"",
```

Comment:
For the Nine Worlds quotes I've been using curvy quotes(“”) for dialog, which you don't have to escape.

Reply:
That's a nice workaround, thanks!